modules/ztp-image-based-upgrade-procedure-steps.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-procedure-steps_{context}"]
+= Performing an image-based upgrade on managed clusters at scale in several steps
+
+For use cases when you need better control of when the upgrade interrupts your service, you can upgrade a set of your managed clusters by using the `ImageBasedGroupUpgrade` CR with adding actions after the previous step is complete.
```

Comment:
I'd avoid "previous" here and call out the section instead. In the event that somehow sections move, "previous" statements can cause issues.

Reply:
It's not referring to a procedure step but a step (one or more actions) in the upgrade plan, so the user will always have to evaluate the previous step in the upgrade plan.